integration/business-process-tests/tests-integration/bpmn/src/test/resources/smokeTestng.xml

```diff
@@ -11,14 +11,15 @@
   ~ Unless required by applicable law or agreed to in writing,
   ~ software distributed under the License is distributed on an
   ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  ~ KIND, either express or implied. See the License for the
+  ~ KIND, either express or implied.  See the License for the
```

Comment:
It seems your idea license template is adding an extra space.

Reply:
done